themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +34,30 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group'] && strtolower($this->config->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'): ?> 
```

Comment:
This doesn't work for me, because `$this->config` is not set in this context. I think you might want to say `$edsConfig = $this->config()->get('EDS');` at the top of the template, and then use `$edsConfig` here....

Reply:
thanks @demiankatz I've added this change to my latest follow-up!